Mammoth/TSE/TSE.vcxproj

```diff
@@ -293,7 +293,6 @@
       <SuppressStartupBanner>true</SuppressStartupBanner>
       <DebugInformationFormat>ProgramDatabase</DebugInformationFormat>
       <CompileAs>Default</CompileAs>
-      <TreatWarningAsError>true</TreatWarningAsError>
```

Comment:
If a particular warning is causing an error on something safe, should probably just suppress that specific warning rather than all